packages/openapi3/src/openapi.ts

```diff
@@ -1046,6 +1046,11 @@ function createOAPIEmitter(program: Program, options: ResolvedOpenAPI3EmitterOpt
         continue;
       }
 
+      if (isNeverType(prop.type)) {
```

Comment:
Wondering if it would make sense to have this built at the checker level, like it doesn't even include the property in the model object then. 

I guess the issue with it then is you can't project it if the property is not there.

Reply:
Yep, I think that's a good way to look at it.  We might want to make the property have some other type in the future, so possibly better to leave it in at the checker level.

Reply:
Should we maybe open a design issue on this? It's also worth considering if failing that it would be good to put it in MetadataInfo.isPayloadProperty called just above. Upside, one fewer thing for an emitter that follows the (to-be-written, #1083) docs on metadata. Seems reasonable to say respond false to both isPayload and isMetadata for that. Speaking of which, do we need to handle `@header something: never`, etc.?

Reply:
Yep, I will open a design issue for it.

Reply:
Oh, I just opened #1128